site/content/extend/plugins/community_process.md

```diff
@@ -31,6 +31,8 @@ Getting your plugin onto our community server https://community.mattermost.com i
 
 Basic code review of an experimental plugin involves a quick review by a [core committer](/contribute/getting-started/core-committers/) to verify that the plugin does what it says it does and to provide any guidance and feedback. To make it easier to provide feedback, a PR can be made that contains all the code of the plugin that isn't the boilerplate from mattermost-plugin-starter-template.
 
+- When you are ready for your plugin to start this process, post an introduction in the ~Toolkit channel on the community server. E.g., "Hi, I have a plugin that I would like to get reviewed. It is called ... It is designed to ... Thanks for reviewing it!"
```

Comment:
Would there be a link for an example post someone has made previously?

Reply:
I don't know, personally -- maybe the one Gabe made for the cloud team's plugin? https://community-release.mattermost.com/core/pl/6dci1ssexjrh9rmdzt4pdpb9zy
But he posted it in `Integrations` -- is that where we want these to go?

Reply:
Depends on owner - if Toolkit team owns reviews, then Toolkit channel makes more sense - otherwise the Integrations channel.

Reply:
I think for now Integrations is owning the process, so I'll make that change and add Gabe's as an example.